cert/spec.go

```diff
@@ -443,9 +433,22 @@ func (spec *Spec) ResetBackoff() {
 // a hint to the invoker as to when the next refresh is required- that said
 // the invoker should back off and try a refresh.
 func (spec *Spec) EnforcePKI(enableActions bool) (time.Duration, error) {
-	err := spec.checkDiskPKI()
+	var certData []byte
+	var keyData []byte
+	var err error
+	certData, err = spec.Cert.ReadFile()
```

Comment:
nit: There's not really a need to declare `certData` and `err`, vs. just using `:=` assignment. I suppose this is just a matter of style and doesn't really matter though, but I usually see it the way I'm saying.